City-of-Helsinki/helsinki-paatokset
# PP-232: Meetings integration updates

This adds new fields and migration details for them.

**To test:**
- Checkout branch
- Run either `make new`, `make fresh` or SSH into the container and run: `composer install;drush cr;drush cim -y`
- Run `drush ap:fs -v`
- Run `drush mim ahjo_meetings:latest --update`
- Check that this content list has meetings in it: https://helsinki-paatokset.docker.so/fi/admin/content/meetings
- Find a meeting with both the agenda and minutes published
- Check that the composition, previous and next meetings, agenda and documents are stored in the node as JSON
